applications/hips/values.yaml

```diff
@@ -10,10 +10,20 @@ config:
   # @default -- None, must be set
   gcsProject: ""
 
-  # -- Name of Google Cloud Storage bucket holding the HiPS files
+  # -- A mapping of dataset names to locations in GCS. /api/hips/v2/<dataset
+  # name> will serve files out of the corresponding GCS bucket.
```

Comment:
same thing here, and splitting it across lines feels even ickier to me.

This is probably just my OCD talking.